generators/setup-docker/templates/fabric-docker/docker-compose.yaml

```diff
@@ -119,7 +119,7 @@ services:<% var rootCaFullName=rootOrg.ca.prefix+"."+rootOrg.organization.domain
     environment:
       - CORE_VM_ENDPOINT=unix:///host/var/run/docker.sock
       - CORE_PEER_ID=<%= peerFullName %>
-      - CORE_LOGGING_PEER=${LOGGING_LEVEL}
+      - FABRIC_LOGGING_SPEC=${LOGGING_LEVEL}
```

Comment:
By the way we could also investigate and add similar flags to `ca`'s. 
We're not yet consistent across all services.

Reply:
fortunately it works with the same env var (fixed)